make/test-integration.mk

```diff
@@ -48,7 +48,7 @@ test-full:
 test-coverage:
 	$(GO) test -tags=$(BUILD_MAINNET) -coverprofile=coverage.txt \
 		-covermode=count \
-		-coverpkg=$(COVER_PACKAGES)
+		-coverpkg=./... $(COVER_PACKAGES)
```

Comment:
i think that needs more significant rework. there used to be incompatibility of `coverprofile` when coverpkg has multiple package. may need to go with loop here

Reply:
nvm, this issue seems fixed

Reply:
i think it should look like that
```make
space := $(subst ,, )
comma := ,

test-coverage:
	$(GO) test -tags=$(BUILD_MAINNET) -coverprofile=coverage.txt \
		-covermode=count \
		-coverpkg=$(subst $(space),$(comma),$(COVER_PACKAGES)) \
		./...
```

Reply:
Did something similar, now the script expands to:
```
GO111MODULE=on go test -tags= -coverprofile=coverage.txt \
        -covermode=count \
        -coverpkg=github.com/ovrclk/akash/app,github.com/ovrclk/akash/app/migrations,github.com/ovrclk/akash/app/params,github.com/ovrclk/akash/client,github.com/ovrclk/akash/client/broadcas
ter,github.com/ovrclk/akash/client/docs/statik,github.com/ovrclk/akash/cmd/akash,github.com/ovrclk/akash/cmd/akash/cmd,github.com/ovrclk/akash/cmd/common,github.com/ovrclk/akash/deploy/cmd,g
ithub.com/ovrclk/akash/docgen,github.com/ovrclk/akash/events,github.com/ovrclk/akash/events/cmd,github.com/ovrclk/akash/integration,github.com/ovrclk/akash/manifest,github.com/ovrclk/akash/p
kg/apis/akash.network/v1,github.com/ovrclk/akash/pkg/client/clientset/versioned,github.com/ovrclk/akash/pkg/client/clientset/versioned/fake,github.com/ovrclk/akash/pkg/client/clientset/versi
oned/scheme,github.com/ovrclk/akash/pkg/client/clientset/versioned/typed/akash.network/v1,github.com/ovrclk/akash/pkg/client/clientset/versioned/typed/akash.network/v1/fake,github.com/ovrclk
/akash/pkg/client/informers/externalversions,github.com/ovrclk/akash/pkg/client/informers/externalversions/akash.network,github.com/ovrclk/akash/pkg/client/informers/externalversions/akash.n
etwork/v1,github.com/ovrclk/akash/pkg/client/informers/externalversions/internalinterfaces,github.com/ovrclk/akash/pkg/client/listers/akash.network/v1,github.com/ovrclk/akash/provider,github
.com/ovrclk/akash/provider/bidengine,github.com/ovrclk/akash/provider/cluster,github.com/ovrclk/akash/provider/cluster/kube,github.com/ovrclk/akash/provider/cluster/types,github.com/ovrclk/a
kash/provider/cluster/util,github.com/ovrclk/akash/provider/cmd,github.com/ovrclk/akash/provider/event,github.com/ovrclk/akash/provider/gateway/rest,github.com/ovrclk/akash/provider/gateway/
utils,github.com/ovrclk/akash/provider/manifest,github.com/ovrclk/akash/provider/session,github.com/ovrclk/akash/provider/testutil,github.com/ovrclk/akash/pubsub,github.com/ovrclk/akash/sdku
til,github.com/ovrclk/akash/sdl,github.com/ovrclk/akash/testutil,github.com/ovrclk/akash/testutil/cli,github.com/ovrclk/akash/testutil/state,github.com/ovrclk/akash/types,github.com/ovrclk/a
kash/types/unit,github.com/ovrclk/akash/util/ctxlog,github.com/ovrclk/akash/util/metrics,github.com/ovrclk/akash/util/runner,github.com/ovrclk/akash/util/validation,github.com/ovrclk/akash/u
til/wsutil,github.com/ovrclk/akash/validation,github.com/ovrclk/akash/validation/constants,github.com/ovrclk/akash/x/audit,github.com/ovrclk/akash/x/audit/client/cli,github.com/ovrclk/akash/
x/audit/client/rest,github.com/ovrclk/akash/x/audit/handler,github.com/ovrclk/akash/x/audit/keeper,github.com/ovrclk/akash/x/audit/query,github.com/ovrclk/akash/x/audit/types,github.com/ovrc
lk/akash/x/cert,github.com/ovrclk/akash/x/cert/client/cli,github.com/ovrclk/akash/x/cert/handler,github.com/ovrclk/akash/x/cert/keeper,github.com/ovrclk/akash/x/cert/simulation,github.com/ov
rclk/akash/x/cert/types,github.com/ovrclk/akash/x/cert/utils,github.com/ovrclk/akash/x/deployment,github.com/ovrclk/akash/x/deployment/client/cli,github.com/ovrclk/akash/x/deployment/client/
rest,github.com/ovrclk/akash/x/deployment/handler,github.com/ovrclk/akash/x/deployment/keeper,github.com/ovrclk/akash/x/deployment/query,github.com/ovrclk/akash/x/deployment/simulation,githu
b.com/ovrclk/akash/x/deployment/types,github.com/ovrclk/akash/x/escrow,github.com/ovrclk/akash/x/escrow/client/cli,github.com/ovrclk/akash/x/escrow/client/rest,github.com/ovrclk/akash/x/escr
ow/keeper,github.com/ovrclk/akash/x/escrow/query,github.com/ovrclk/akash/x/escrow/types,github.com/ovrclk/akash/x/market,github.com/ovrclk/akash/x/market/client/cli,github.com/ovrclk/akash/x
/market/client/rest,github.com/ovrclk/akash/x/market/handler,github.com/ovrclk/akash/x/market/hooks,github.com/ovrclk/akash/x/market/keeper,github.com/ovrclk/akash/x/market/query,github.com/
ovrclk/akash/x/market/simulation,github.com/ovrclk/akash/x/market/types,github.com/ovrclk/akash/x/provider,github.com/ovrclk/akash/x/provider/client/cli,github.com/ovrclk/akash/x/provider/cl
ient/rest,github.com/ovrclk/akash/x/provider/config,github.com/ovrclk/akash/x/provider/handler,github.com/ovrclk/akash/x/provider/keeper,github.com/ovrclk/akash/x/provider/query,github.com/o
vrclk/akash/x/provider/simulation,github.com/ovrclk/akash/x/provider/types \
        ./...
```